QualityControl/lib/controllers/ObjectController.js

```diff
@@ -79,57 +65,31 @@ export class ObjectController {
    * @returns {void}
    */
   async getObjectContent(req, res) {
-    const { path, validFrom, id, filters } = req.query;
-    if (!path) {
-      updateAndSendExpressResponseFromNativeError(
-        res,
-        new InvalidInputError('Invalid URL parameters: missing object path'),
-      );
-    } else {
-      try {
-        const object = await this._objService.retrieveQcObject(path, Number(validFrom), id, filters);
-        res.status(200).json(object);
-      } catch (error) {
-        updateAndSendExpressResponseFromNativeError(
-          res,
-          new Error('Unable to identify object or read it'),
-        );
-        this._logger.errorMessage(error);
-      }
+    try {
+      const { path, validFrom, filters, id } = req.query;
+
+      const object = await this._objService.retrieveQcObject(path, validFrom, id, filters);
+      res.status(200).json(object);
+    } catch (error) {
+      const responseError = new Error('Failed to retrieve object content');
+
+      logger.errorMessage(`Error validating query parameters: ${error}`);
```

Comment:
There is no error to validate the query here. The input parameters are evaluated in the middleware now